glue-codes/labview/src/WaveTank.f90

```
@@ -0,0 +1,228 @@
+
+MODULE WaveTankTesting
+
+    USE ISO_C_BINDING
+    USE NWTC_Library
+    ! USE Precision
+    USE MoorDyn_C
+    USE SeaState_C_Binding
+    USE NWTC_C_Binding, ONLY: IntfStrLen, SetErr
+
+    IMPLICIT NONE
+    SAVE
+
+    PUBLIC :: WaveTank_Init
+
+    REAL(C_DOUBLE) :: dt_c = 0.01  ! 100 hertz
+    REAL(C_FLOAT) :: g_c = 9.8065
+    REAL(C_FLOAT) :: rho_c = 1025
+    REAL(C_FLOAT) :: depth_c = 200
+    REAL(C_FLOAT), DIMENSION(6) :: ptfminit_c = (/ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0/)
```

```suggestion
    REAL(C_DOUBLE) :: dt_c = 0.01_C_DOUBLE  ! 100 hertz
    REAL(C_FLOAT) :: g_c = 9.8065_C_FLOAT
    REAL(C_FLOAT) :: rho_c = 1025.0_C_FLOAT
    REAL(C_FLOAT) :: depth_c = 200.0_C_FLOAT
    REAL(C_FLOAT), DIMENSION(6) :: ptfminit_c = 0.0_C_FLOAT
```

Type specifiers needed on floating point constants; otherwise, they may have they may default to the wrong type when compiling in single or double precision.

---

Right good catch, I'll make this change